apps/api/e2e/tests/competition.test.ts

```diff
@@ -481,17 +481,17 @@ describe("Competition API", () => {
     const createResponse1 = (await adminClient.createCompetition(
       comp1Name,
       "Test competition 1",
-      CrossChainTradingType.allow,
+      CROSS_CHAIN_TRADING_TYPE.ALLOW,
```

Comment:
here's how the pattern changes w/o the TS enum type, and with `as const` type instead. (fwiw, i personally dont have a strong preference b/w the two)